webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var path = require("path");
```

Comment:
Do we still need webpack config to solve this issue after using HashRouter. If yes, why? If not, we should clean up this file and npm dependencies.

Reply:
Yes this was redundant after HashRouter, I've removed that and updated dependencies.